pkg/client/apiutil/apimachinery_test.go

```diff
@@ -18,6 +18,7 @@ package apiutil_test
 
```

Comment:
nit for the PR description. Based on Kubernetes docs this is enabled per default since 1.27: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-graduated-or-deprecated-features

Reply:
Updated